resources/fixtures/Clay_Paky/Clay-Paky-Mini-Scan-HPE.qxf

```diff
@@ -66,7 +60,7 @@
  </Channel>
  <Mode Name="Default">
   <Channel Number="0">Color Disc</Channel>
-  <Channel Number="1">Gobo Rotation</Channel>
+  <Channel Number="1" ActsOn="2">Gobo Rotation</Channel>
```

Comment:
ActsOn="0"?

Reply:
ActsOn="2" is correct, as the gobo selection here is on channel 2 (it's a strange device)